microshift_updating/microshift-about-updates.adoc

```diff
@@ -0,0 +1,46 @@
+:_content-type: ASSEMBLY
+[id="microshift-about-updates"]
+=  About {product-title} updates
```

Comment:
I know you're thinking about this, but is this assembly going to be specifically for information about updating, and then a separate assembly for updating? If this assembly is solely for information, I think the title works. If it's not, maybe just "Updating {product-title}" could be generic enough to encompass information about it + the procedures.